docker/initial-data.json

```diff
@@ -14,6 +14,8 @@
             "committee_ids": [
                 1
             ],
+            "active_meetings": [1],
+            "limit_of_meetings": 3,
```

Comment:
Please set "no limit" for the initial data.

Reply:
Done, set to 0 (=no_limit